frontend/app/routes/public/apply/$id/adult/phone-number.tsx

```diff
@@ -88,11 +87,8 @@ export async function action({ context: { appContainer, session }, params, reque
   // TODO: Uncomment when address routes are available
   // saveApplyState({ params, session, state: { contactInformation: { ...state.contactInformation, ...parsedDataResult.data } } });
 
-  const idToken: IdToken = session.get('idToken');
-  appContainer.get(TYPES.domain.services.AuditService).createAudit('update-data.renew.phone-number', { userId: idToken.sub });
-
   if (state.editMode) {
-    return redirect(getPathById('public/apply/$id/review-adult-information', params));
+    return redirect(getPathById('public/apply/$id/adult/review-adult-information', params));
```

Comment:
Since adult flow will only have a single review page, and there's no defined route ID for `public/apply/$id/adult/review-adult-information`:

```suggestion
    return redirect(getPathById('public/apply/$id/adult/review-information', params));
```

(same comment for the `routeId` for the Cancel button in this file)